.github/workflows/render-proof-gate.yml

```
@@ -0,0 +1,85 @@
+jobs:
+  render-proof-check:
+    name: Check for render proof in PR description
+    runs-on: ubuntu-latest
+    continue-on-error: true
+    permissions:
+      pull-requests: write
+
+    steps:
+      - name: Detect milestone from branch name
+        id: milestone
+        run: |
```

actionlint (1.7.12)

[error] "github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)